### PR TITLE
Add meta for E&F theme bundles

### PR DESCRIPTION
**Pull request by richardhj**

### CODE REVIEW

File: meta/erdmannfreunde/lasr-theme-bundle/en.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+de:
+    title: SIRIUS theme bundle
```
Comment (xchs) [inline meta/erdmannfreunde/lasr-theme-bundle/en.yml]:
```suggestion
    title: LASR theme bundle
```

File: meta/erdmannfreunde/lasr-theme-bundle/en.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+    description: Install all extensions and dependencies for the SIRIUS theme by Erdmann & Freunde.
```
Comment (xchs) [inline meta/erdmannfreunde/lasr-theme-bundle/en.yml]:
```suggestion
    description: Install all extensions and dependencies for the LASR theme by Erdmann & Freunde.
```

File: meta/erdmannfreunde/sirius-theme-bundle/de.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+de:
+    title: SOLO-Theme-Bundle
```
Comment (xchs) [inline meta/erdmannfreunde/sirius-theme-bundle/de.yml]:
```suggestion
    title: SIRIUS-Theme-Bundle
```

File: meta/erdmannfreunde/sirius-theme-bundle/de.yml [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+    description: Installiere mit einem Klick alle Erweiterungen für das Erdmann-&-Freunde-Theme "SOLO".
```
Comment (xchs) [inline meta/erdmannfreunde/sirius-theme-bundle/de.yml]:
```suggestion
    description: Installiere mit einem Klick alle Erweiterungen für das Erdmann-&-Freunde-Theme "SIRIUS".
```

File: meta/erdmannfreunde/sirius-theme-bundle/de.yml [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+    keywords:
+        - SOLO
```
Comment (xchs) [inline meta/erdmannfreunde/sirius-theme-bundle/de.yml]:
```suggestion
        - SIRIUS
```

File: meta/erdmannfreunde/sirius-theme-bundle/en.yml [hunk 6]
```diff
@@ -0,0 +1,7 @@
+de:
+    title: SIRIUS theme bundle
+    description: Install all extensions and dependencies for the SIRIUS theme by Erdmann & Freunde.
+    keywords:
+        - SOLO
```
Comment (xchs) [inline meta/erdmannfreunde/sirius-theme-bundle/en.yml]:
```suggestion
        - SIRIUS
```

File: meta/erdmannfreunde/lasr-theme-bundle/en.yml [hunk 7]
```diff
@@ -0,0 +1,7 @@
...
+    keywords:
+        - SOLO
```
Comment (xchs) [inline meta/erdmannfreunde/lasr-theme-bundle/en.yml]:
```suggestion
        - LASR
```

**Comment by richardhj** [comment]
Thank you @xchs 😄

**Pull request merged**